Sources/SWBUtil/fnmatch.swift

```diff
@@ -110,14 +100,14 @@ public func fnmatch(pattern: some StringProtocol, input: some StringProtocol, op
             bt_pattern = pattern
             bt_input = input
         case "[":
-            guard let _sc = input.first else {
```

Comment:
Reordering these checks lets us do a quick utf8 check without having to construct a Character (since those are String under the hood)